src/renderers/shared/shared/__tests__/ReactCompositeComponentState-test.js

```diff
@@ -171,39 +171,20 @@ describe('ReactCompositeComponent-state', () => {
       ['componentDidMount-end', 'orange'],
       ['setState-sunrise', 'orange'],
       ['setState-orange', 'orange'],
-    ];
-
-    // In Fiber, the initial callback is not enqueued until after any work
-    // scheduled by lifecycles has flushed (same semantics as a regular setState
-    // callback outside of a batch). In Stack, the initial render is scheduled
-    // inside of batchedUpdates, so the callback gets flushed right after
-    // componentDidMount.
-    // TODO: We should fix this, in both Stack and Fiber, so that the behavior
-    // is consistent regardless of whether you're in a batch.
-    if (!ReactDOMFeatureFlags.useFiber) {
-      expected.push(['initial-callback', 'orange']);
-    }
-
-    expected.push(['shouldComponentUpdate-currentState', 'orange'],
+      ['initial-callback', 'orange'],
+      ['shouldComponentUpdate-currentState', 'orange'],
       ['shouldComponentUpdate-nextState', 'yellow'],
       ['componentWillUpdate-currentState', 'orange'],
       ['componentWillUpdate-nextState', 'yellow'],
       ['render', 'yellow'],
       ['componentDidUpdate-currentState', 'yellow'],
       ['componentDidUpdate-prevState', 'orange'],
       ['setState-yellow', 'yellow'],
-    );
-
-    if (ReactDOMFeatureFlags.useFiber) {
-      expected.push(['initial-callback', 'yellow']);
```

Comment:
Explained by the now-deleted comment above:

> In Fiber, the initial callback is not enqueued until after any work scheduled by lifecycles has flushed (same semantics as a regular setState callback outside of a batch). In Stack, the initial render is scheduled inside of batchedUpdates, so the callback gets flushed right after componentDidMount.
>
> TODO: We should fix this, in both Stack and Fiber, so that the behavior is consistent regardless of whether you're in a batch.